fix(compiler): reduce chance of conflicts between generated factory and local variables

crisbeto opened the pull request:
Currently we use some short variable names like `t` and `r` in the generated factory functions. They can conflict with local symbols with the same names, if they're used for DI.

These changes rename the parameters to reduce the change for conflicts.

Fixes #57168.

dylhunn commented:
This PR was merged into the repository by commit 67e09404db4a8a3a09bff005503a76f49d4fe055.

The changes were merged into the following branches: main, 18.1.x

crisbeto committed:
fix(compiler): reduce chance of conflicts between generated factory and local variables

Currently we use some short variable names like `t` and `r` in the generated factory functions. They can conflict with local symbols with the same names, if they're used for DI.

These changes rename the parameters to reduce the change for conflicts.

Fixes #57168.

crisbeto commented:
I've changed the naming scheme so the parameters don't contain any special characters. I've also [run a TGP which is passing](http://test/OCL:657181935:BASE:660396135:1723043736535:3a619774).

thePunderWoman commented:
This PR was merged into the repository by commit d9d68e73d2b59b598d1f7de03ad5faa2b6d31ee2.

The changes were merged into the following branches: main, 18.1.x
